base/fhir_source_connect_processor.bal

```diff
@@ -101,13 +101,10 @@ public isolated function processFHIRSourceConnections(FHIRSourceConnectInteracti
         return;
     } else if fhirContext.getInteraction().interaction.equalsIgnoreCaseAscii(UPDATE) {
         //respond 200 if updated, 201 with location header if created
```

Comment:
Shall we create issue to track the binding of relevant functions for update and delete interactions?

Reply:
these are not implemented yet in the templates or processors. Will create an issue to track.